packages/viewer/src/components/scene-tree/scene-tree.tsx

```diff
@@ -972,7 +978,9 @@ export class SceneTree {
   }
 
   private getConfig(): Config {
-    return parseConfig(this.configEnv, this.config);
+    const parsedConfig = parseAndValidateConfig(this.configEnv, this.config);
+
+    return validateConfig(sanitizeConfig(parsedConfig));
```

Comment:
It looks like `validateConfig` and `sanitizeConfig` are run as part of `parseAndValidateConfig`, so are they needed here too?

Reply:
Ah good catch. I pulled those two into `parseAndValidateConfig` after realizing we had quite a few places we needed to do this validation + sanitation, but missed removing those here. Updated!